### ISSUE TITLE
13.11.6 (11/4) Again please move record to align with end record 

### ISSUE THREAD

**Issue by pat-rogers** [pat-rogers/Ada-202x-WG9-Informal-Review#70]
Again please move **record** to align with **end record** so it looks more like:

```ada
type Mark_Release ....    ....) is new 
  Subpools.Root_Storage_Pool_With_Subpools with
  record
     Storage....
     Next_Allocation.....
     Markers......
     Current_Pool......
  end record;```

**Comment by ARG-Editor** [comment]
That is incorrect Ada RM style, at least based on the grammar of record_extension (see 3.9.1) -- there is no line break before "record_definition". This is not the place to argue correctness of that style - I personally am opposed to the style you want since it wastes space both horizontally (extra level of indention) and vertically, See also my response to #68.
